### ISSUE TITLE
Diaspora: Image URLs with double "http://" throw BBCode off

### ISSUE THREAD

**Issue by MrPetovan** [friendica/friendica#4451]
Original post: https://joindiaspora.com/posts/db041f00f2e801358c0e0242ac110003
Friendica display: https://friendica.mrpetovan.com/display/db041f00f2e801358c0e0242ac110003
Source:
````xml
<status_message>
  <author>hernan_labbe@joindiaspora.com</author>
  <guid>db041f00f2e801358c0e0242ac110003</guid>
  <created_at>2018-02-13T12:39:28Z</created_at>
  <public>true</public>
  <text>![](https://web.archive.org/web/20091019235542/http://www.geocities.com/wonder-of-god@sbcglobal.net/images/checkit1.gif)&#xD;
&#xD;
Holy smokes! We reached the 10,000 views!!! (that is, when youtube is not removing views because gods know why). &#xD;
&#xD;
I guess the video must be good... Or maybe the song rocks? Probably both.... But you'll have to see to find out! &#xD;
&#xD;
So check our our new video! from our latest single,  [Intergalactic Pie](https://www.youtube.com/watch?v=wwUsYpMn_f0) ![](https://web.archive.org/web/20091025034815/http://www.geocities.com/fazedoubt/banana2.gif)&#xD;
&#xD;
And share it like you mean it! ;)&#xD;
&#xD;
Check out more of our music [HERE](https://lolasdice.bandcamp.com/music)&#xD;
&#xD;
#SupportDiasporaArtists #music #video #mymusic #mywork #myvideo #videoclip #muziek #Nederland #Netherlands #nl #funk #psychedelic #rock #animation #animated #lolasdice #IntergalacticPie</text>
</status_message>
````

**Comment by MrPetovan** [comment]
Another example, this time the whole title + text has been truncated from a post with an inline image:
Original post: https://diaspora-fr.org/posts/535d72b0f5aa0135d6763b18ba1c589f
Friendica display: https://friendica.mrpetovan.com/display/535d72b0f5aa0135d6763b18ba1c589f
Object: 
````xml
<status_message>
  <author>salinger3@diaspora-fr.org</author>
  <guid>535d72b0f5aa0135d6763b18ba1c589f</guid>
  <created_at>2018-02-17T00:49:25Z</created_at>
  <public>true</public>
  <text>## Maladie de Lyme: 59 patients portent plainte et d&#xE9;noncent des collusions&#xD;
&#xD;
![](https://s.yimg.com/ny/api/res/1.2/q73RqXFr475P_ns9J.v3CQ--/YXBwaWQ9aGlnaGxhbmRlcjtzbT0xO3c9NzY4O2g9NTExO2lsPXBsYW5l/http://media.zenfs.com/fr_FR/News/AFP/e8d1ab8e3fbe742a79f7906298e9f0c1f83c505e.jpg)&#xD;
&#xD;
Paris (AFP) - Une soixantaine de personnes atteintes de la #maladie de #Lyme ont d&#xE9;pos&#xE9; une plainte jeudi &#xE0; Paris, qui d&#xE9;nonce de possibles conflits d'int&#xE9;r&#xEA;ts entre les autorit&#xE9;s sanitaires et les laboratoires, a-t-on appris aupr&#xE8;s de leurs avocats et de source judiciaire.&#xD;
&#xD;
"59 plaignants sont d&#xE9;j&#xE0; enregistr&#xE9;s sur 152 malades pour cette premi&#xE8;re vague", ont d&#xE9;clar&#xE9; &#xE0; l'AFP les avocats Julien Fouray et Catherine Faivre, qui ont d&#xE9;pos&#xE9; cette plainte contre X au p&#xF4;le sant&#xE9; publique du parquet de Paris.&#xD;
&#xD;
Selon leurs conseils, **les plaignants et l'association Lyme Sans Fronti&#xE8;re r&#xE9;clament une enqu&#xEA;te visant notamment l'Agence du m&#xE9;dicament (ANSM) et le Centre national de r&#xE9;f&#xE9;rence de Strasbourg, dont ils pointent le manque de vigilance face &#xE0; des tests jug&#xE9;s non fiables.**&#xD;
&#xD;
Cette plainte, dont le projet avait &#xE9;t&#xE9; annonc&#xE9; mi-d&#xE9;cembre, d&#xE9;nonce de possibles faits de "conflits d'int&#xE9;r&#xEA;ts", "violation des r&#xE8;gles d'ind&#xE9;pendance en mati&#xE8;re d'expertise", "trafic d'influence", "abus de biens sociaux et recel", "corruption" et "mise en danger de la vie d'autrui", ont d&#xE9;taill&#xE9; Mes Fouray et Faivre.&#xD;
&#xD;
Les avocats, qui comptent r&#xE9;unir &#xE0; terme environ 300 plaignants, ont jusqu'ici entam&#xE9; uniquement des proc&#xE9;dures en responsabilit&#xE9; civile contre cinq laboratoires (DiaSorin, BioM&#xE9;rieux, Siemens, Euroimmun et Bio-Rad) afin d'obtenir des dommages et int&#xE9;r&#xEA;ts. Ils les accusent de ne pas avoir suffisamment inform&#xE9; le public sur le manque de fiabilit&#xE9; des tests de diagnostic qu'ils commercialisent.&#xD;
&#xD;
Ces proc&#xE9;dures, toujours en cours, "ont permis de r&#xE9;unir des &#xE9;l&#xE9;ments sur les possibles conflits d'int&#xE9;r&#xEA;ts et collusions entre des responsables des autorit&#xE9;s sanitaires et les fabricants", a expliqu&#xE9; Me Faivre.&#xD;
&#xD;
Cette plainte p&#xE9;nale vient s'ajouter &#xE0; celle d&#xE9;pos&#xE9;e &#xE0; Paris en f&#xE9;vrier 2017 par une malade r&#xE9;sidant dans le Cantal. D&#xE9;fendue par un autre avocat, Philippe Meilhac, elle avait port&#xE9; plainte, &#xE9;galement &#xE0; Paris, mais pour "tromperie aggrav&#xE9;e".&#xD;
&#xD;
Sa plainte avait d&#xE9;bouch&#xE9; sur une enqu&#xEA;te pr&#xE9;liminaire de ce p&#xF4;le sp&#xE9;cialis&#xE9; du parquet de Paris. Elle est toujours en cours, selon une source proche du dossier.&#xD;
&#xD;
**La maladie de Lyme, infection transmise par les tiques, peut provoquer de nombreux troubles invalidants et douloureux.**&#xD;
&#xD;
**D'un diagnostic difficile, elle a longtemps &#xE9;t&#xE9; sous-estim&#xE9;e et le minist&#xE8;re de la Sant&#xE9; a relev&#xE9; en 2016 son estimation du nombre de cas en France &#xE0; 55.000, contre 33.000 l'ann&#xE9;e pr&#xE9;c&#xE9;dente.**&#xD;
&#xD;
**Des associations de malades et certains professionnels de sant&#xE9; militent pour la reconnaissance de la forme chronique de cette maladie, une sensibilisation de la population et la modification du protocole officiel de diagnostic et de soins**&#xD;
&#xD;
#sant&#xE9; </text>
</status_message>
````

**Comment by MrPetovan** [comment]
I see a pattern: in both cases, the images URL include `http://` twice, it must be throwing our fantastic URL pattern matching off.

**Comment by MrPetovan** [comment]
An intermediate solution would be to escape the BBCode possibly containing unwarranted URLs before we run the autolinker regular expression. We already do it for code blocks before BBCode replacement.

**Comment by MrPetovan** [comment]
Fixed with #7944
